### ISSUE TITLE
rehydrate only works when devtools intrument enabled?

### ISSUE THREAD

**Issue by creat-or** [btroncone/ngrx-store-localstorage#72]
I try to save the authentication information inside localStorage. Now, in development everything worked fine, but in production the information is not rehydrated.

In my root module imports I have the following line: 
```typescript
!environment.production ? StoreDevtoolsModule.instrument() : [],
```

When I change it to instrument in development as well as in production, suddenly also rehydration of the state works.
```typescript
StoreDevtoolsModule.instrument(),
```

Do you have an idea what could cause the problem?

For completeness, here is how I activate ngrxLocalStorage (based on the ngrx example project in `app/reducers/index.ts`):

```typescript
export function localStorageSyncReducer(reducer: ActionReducer<any>): ActionReducer<any> {
  return localStorageSync({ keys: ['auth'], rehydrate: true })(reducer);
}

export const metaReducers: ActionReducer<any, any>[] = !environment.production
  ? [localStorageSyncReducer, logger]
  : [localStorageSyncReducer];
```

**Comment by chocmah** [comment]
Same problem here, it's working fine only when devtool are enabled

 "@angular/animations": "^5.0.1",
    "@angular/common": "^5.0.1",
    "@angular/compiler": "^5.0.1",
    "@angular/core": "^5.0.1",
    "@angular/forms": "^5.0.1",
    "@angular/http": "^5.0.1",
    "@angular/platform-browser": "^5.0.1",
    "@angular/platform-browser-dynamic": "^5.0.1",
    "@angular/router": "^5.0.1",
    "@ngrx/effects": "^4.1.1",
    "@ngrx/router-store": "^4.1.1",
    "@ngrx/store": "^4.1.1",
    "@ngrx/store-devtools": "^4.1.1",
    "ngrx-store-localstorage": "^0.2.4",

**Comment by chocmah** [comment]
I fixed it by moving the meta-reducer into the specific feature module of where i wanted to persist the state

`        StoreModule.forFeature('auth', reducers, { metaReducers: metaReducers }),
` 

`export const metaReducers: MetaReducer<any, Action>[] = [localStorageSyncReducer];`

```
export function localStorageSyncReducer(reducer: ActionReducer<any>): ActionReducer<any> {
    return localStorageSync({ keys: ['status'], rehydrate: true })(reducer);
}
```

**Comment by creat-or** [comment]
Thank you! That also worked for me.

If this is the intended behaviour, feel free to close this issue :)

**Comment by glebcha** [comment]
Moved to specific module with no result. It can only work as declared for root.

**Comment by un33k** [comment]
@chocmah perhaps you show where you use `localStorageSyncReducer`.

**Comment by tanyagray** [comment]
This issue should be resolved as of `v5.0.0` thanks to @nhaesler's PR 👍
